### PR TITLE
Iss1250 1

### PR DESCRIPTION
**Pull request by kelseamann**
closes issue 1250 from patternfly-design

### CODE REVIEW

**Comment by nicolethoen** [comment]
@kelseamann did you intend to add images as part of this PR? I don't see an image named 'primary-button' in the toolbar/img directory.

**Comment by nicolethoen** [comment]
yeah - add it anyway. If anything weird happens, i'll clean it up :)

**Comment by kelseamann** [comment]
@nicolethoen I added the image; I set this one up with @mmenestr the other day, so I think I'm on the right branch and all, but if you want to take a look together or have me start over, I'm around!

**Comment by mmenestr** [comment]
@nicolethoen we have tried everything and it is still failing, please help haha

**Comment by nicolethoen** [comment]
It's complaining that it's missing `greater-than-three.png` and `three-as-a-group.png`. Can you add those to the PR?

File: packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md [hunk 1]
```diff
@@ -37,7 +37,22 @@ A search filter is intended to be used with a filter group as shown below when y
...
+* Use a primary button for any call to action you want a user to take
+* In the case that using the action represented should not be encouraged, opt for a secondary button
+
+ **Three clickable actions**
+<img src= "./img/three-as-a-group.png" alt= "three actions" width="196"/>
+ * In the specific case where three actions are needed, the suggested guidance is to opt for an overflow menu with two items. This will prevent the toolbar from becoming too crowded, and also prevent the overflow menu from containing just one item. 
+
+ **More than three clickable actions**
+ <img src= "./img/greater-than-three.png" alt= "primary and secondary button" width="196"/>
+1. Use a primary button to indicate the most important action within the table
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md:53]: Also this is a numbered list, but the other ones are just bullet points. Make this a bulleted list to match!

File: packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md [hunk 2]
```diff
@@ -37,7 +37,22 @@ A search filter is intended to be used with a filter group as shown below when y
...
- Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. No more than two items should be exposed as buttons. If you have more than two items, use an [overflow menu](#overflow-menu) component to save space.
+Any global actions that apply to selected data items are included left-aligned following filter and icon buttons. Be selective about which actions to surface on the toolbar, exposing no more than two buttons and reserving the rest for an [overflow menu](#overflow-menu) component to save space.
+
+ **One or two clickable actions**
+ <img src= "./img/primary-button.png" alt= "single button" width="196"/>
+* Use a primary button for any call to action you want a user to take
+* In the case that using the action represented should not be encouraged, opt for a secondary button
+
+ **Three clickable actions**
+<img src= "./img/three-as-a-group.png" alt= "three actions" width="196"/>
```
Comment (mmenestr) [inline packages/v4/patternfly-docs/content/design-guidelines/components/toolbar/toolbar.md:48]: add line of space between bolded text and images in all these images, and update width to be correct

**Pull request merged**